lib/pandadoc/api/document.rb

```diff
@@ -36,6 +36,7 @@ def create(token, params = {})
           fields: { required: false, type: Hash },
           metadata: { required: false, type: Hash },
           pricing_tables: { required: false, type: Array },
```

Comment:
🙋 Should we remove this? Normally I'd leave it for backwards compatibility, but I don't think that pricing tables will be usable as we've been using them after PandaDoc changes over their API 🤔

Reply:
I think we should leave it in.

This is a public gem and should still support pricing tables.